Makefile

```diff
@@ -96,8 +96,8 @@ endif
 #
 # Example:
 #   make check-test
-check-test: export KUBE_COVER= -cover -covermode=atomic
-check-test: export KUBE_RACE=  -race
+check-test: export COVERAGE_SPEC=-cover -covermode=atomic
+check-test: export DETECT_RACES=true
```

Comment:
Maybe there was a good reason we used the `KUBE_` prefix? It might be for trying to keep ourselves close to upstream:
https://github.com/kubernetes/kubernetes/blob/master/hack/test-go.sh has KUBE_TIMEOUT, KUBE_RACE, KUBE_COVER, etc.
Are we sure not to break anything of importance with this change?